interfaces/cython/cantera/thermo.pyx

```diff
@@ -632,7 +632,52 @@ cdef class ThermoPhase(_SolutionBase):
         stoichAirFuelRatio = - (Of - 2*Cf - 2*Sf - Hf/2.0) / (Oo - 2*Co - 2*So - Ho/2.0)
         Xr = phi * Xf + stoichAirFuelRatio * Xo
         self.TPX = None, None, Xr
+        
+    def get_equivalence_ratio(self, oxidizers=[], ignore=[]):
+        """Get the composition of a fuel/oxidizer mixture.
+        Considers the oxidation of C to CO2, H to H2O and S to SO2.
```

Comment:
Maybe the extra bit of clarity that would be useful to add here would be to say that this is the equivalence ratio of an _unburned_ fuel/oxidizer mixture, to distinguish this definition from one which is conserved after combustion.